### PR TITLE
0001-feat-add-algolia_get_post_object_id-filter

### PR DESCRIPTION
**Pull request by sebastianthulin**
Previously discussed here: https://github.com/algolia/algoliasearch-wordpress/pull/821

### CODE REVIEW

**Comment by richaber** [comment]
Howdy @sebastianthulin.

Could you provide more information about the use case?  
The referenced discussion is a little hard to follow.

If I understand correctly, you have a multisite install, and all of the subsites use the same index?
So there are collisions between object IDs (post ID, term ID, user ID)?

**Comment by sebastianthulin** [comment]
Yes. This is a frequently requested feature that multiple persons has tried to develop in algolia wordpress plugin. It's not a "fancy" solution, but it works with minimal changes to the plugin. 

Here is a usage-example: https://github.com/helsingborg-stad/algolia-multisite-enabler/blob/master/algoliaMultisiteEnabler.php. Have a look at row L40 & L86.
